[BUG] Syntax error when connection failure in "wait_service_ready"

**Describe the bug**

The main branch CI ( https://github.com/alibaba/GraphScope/runs/3629239835#step:10:2177 ) has catched the following unexpected exception,

```python
    def waiting_service_ready(self, timeout_seconds=60):
        begin_time = time.time()
        request = message_pb2.HeartBeatRequest()
        while True:
            code = self._launcher.poll()
            if code is not None and code != 0:
                raise RuntimeError(f"Start coordinator failed with exit code {code}")
            try:
                self._stub.HeartBeat(request)
                logger.info("GraphScope coordinator service connected.")
                break
            except grpc.RpcError as e:
                # Cannot connect to coordinator for a short time is expected
                # as the coordinator takes some time to launch
                msg = f"code: {e.code().name}, details: {e.details()}"
                if e.code() == grpc.StatusCode.DEADLINE_EXCEEDED:
                    logger.warning("Heart beat analytical engine failed, %s", msg)
                else:
                    logger.warning("Heart beat coordinator failed, %s", msg)
                if time.time() - begin_time >= timeout_seconds:
                    raise ConnectionError(
                        "Connect coordinator timeout, coordinator code: %s, details: %s",
                        e.code().name,
>                       e.details(),
                    )
E                   TypeError: __init__() takes from 1 to 3 positional arguments but 4 were given
```
]

**Additional context**

Add any other context about the problem here.

--- a/python/graphscope/client/rpc.py
+++ b/python/graphscope/client/rpc.py
@@ -124,11 +124,7 @@ class GRPCClient(object):
                 else:
                     logger.warning("Heart beat coordinator failed, %s", msg)
                 if time.time() - begin_time >= timeout_seconds:
-                    raise ConnectionError(
-                        "Connect coordinator timeout, coordinator code: %s, details: %s",
-                        e.code().name,
-                        e.details(),
-                    )
+                    raise ConnectionError(f"Connect coordinator timeout, {msg}")
                 time.sleep(1)
 
     def connect(self, cleanup_instance=True, dangling_timeout_seconds=60):

